cdap-distributions/src/.dockerignore

```diff
@@ -0,0 +1,24 @@
+#
```

Comment:
Correct. A docker context is created from the Dockerfile's directory and subdirectories. Normally, a Dockerfile would be at the top of a repository. However, since ours is a bit special use (and doesn't actually require the code in cdap-standalone/etc.), I've put it into the most appropriate place I know. Adding these things to .dockerignore simply means they don't get sent to the Docker daemon during the build as part of the context, speeding up the build, since we don't use any of it.